theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveEditorPopover.tsx

```diff
@@ -370,6 +382,9 @@ const CurveEditorPopover: React.FC<IProps> = (props) => {
             )}
           />
         ))}
+        {displayedPresets.length === 0 ? (
+          <NoResultsFoundContainer>No results found</NoResultsFoundContainer>
+        ) : undefined}
```

Comment:
For @AriaMinaei

Consider the following just in case there is some oddity with typing in something someone expects to work, but it doesn't for whatever reason.
```suggestion
        {displayedPresets.length === 0 ? (
          <NoResultsFoundContainer>No results found for "{filter}".</NoResultsFoundContainer>
        ) : undefined}
```

Also consider when there are no results, to also follow the "No results found" with the list of all the other results.

Reply:
Playing it out in my head, I think `"no results found for ${filter}"` would help. @vezwork @mariusz-wnek-theatre please try that.

Not sure about showing a list of all other results. It's your call @mariusz-wnek-theatre @vezwork

Reply:
Resolved to not take any action on the `No results found for "{...}"` here, as it initially feels very redundant.
![image](https://user-images.githubusercontent.com/2925395/166263924-db08660f-74d5-4b0e-87da-cdb990ff7d52.png)